sklearn/neural_network/tests/test_mlp.py

```diff
@@ -59,7 +60,7 @@ def test_alpha():
     absolute_sum = lambda x: np.sum(np.abs(x))
 
     for alpha in alpha_values:
-        mlp = MLPClassifier(hidden_layer_sizes=10, alpha=alpha, random_state=1)
+        mlp = MLPClassifier(hidden_layer_sizes=10, alpha=alpha, random_state=1, max_iter=500)
```

Comment:
> > +        mlp = MLPClassifier(hidden_layer_sizes=10, alpha=alpha, random_state=1, max_iter=500)

The default of max_iter is 200. By changing the value to 500, haven't you
made the test 2.5 times longer? That would be a problem, as we are trying
to keep the tests as fast as possible.